### DISCUSSION TITLE
求战双

### DISCUSSION THREAD

**Discussion by AthenaRoland** [Gamuxorg/bbs#420] [投递和请求游戏]
官网网址：https://pns.kurogame.com/#Home
求打包android版的
模拟器版也可以

**Comment by hyh19962008** [comment]
网游更新非常频繁，不好打包

Best
________________________________
From: Roland ***@***.***>
Sent: Thursday, August 31, 2023 1:31:29 PM
To: Gamuxorg/bbs ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [Gamuxorg/bbs] 求战双 (Discussion #420)


官网网址：https://pns.kurogame.com/#Home
求打包android版的
模拟器版也可以

―

**Comment by Edward-EH-Holmes** [comment]
> 官网网址：https://pns.kurogame.com/#Home 求打包android版的 模拟器版也可以

给你指条明路，lutris搞wine，直接PC版，自己补个字体就完事了。我这边在manjaro上已经用了几个月了，没啥问题。
